cmd/operator-sdk/olm/status.go

```diff
@@ -35,6 +35,7 @@ func newStatusCmd() *cobra.Command {
 	}
 
 	cmd.Flags().StringVar(&mgr.OLMNamespace, "olm-namespace", olm.DefaultOLMNamespace, "namespace where OLM is installed")
+	cmd.Flags().StringVar(&mgr.Version, "version", "", "version of OLM installed on cluster")
```

Comment:
Should we describe what happens when the flag is unset?
```suggestion
	cmd.Flags().StringVar(&mgr.Version, "version", "", "version of OLM installed on cluster; if unset, operator-sdk attempts to auto-discover the version")
```